src/commands/client.ts

```diff
@@ -291,6 +280,17 @@ export class CommandClient extends Client implements CommandClientOptions {
       guild: msg.guild
     }
 
+    const isUserBlacklisted = await this.isUserBlacklisted(msg.author.id)
+    if (isUserBlacklisted) return this.emit('commandUserBlacklisted', ctx)
```

Comment:
pretty good work but I'm not sure if users want the whole ctx in the blacklist event...
I personally think that it's good to move back to the top and give the user the appropriate object (like give `User` for the event `commandUserBlacklisted` and so on)

Reply:
I thought that it would be nice to give the ctx as you might want to reply to the user and inform them that they're blacklisted.

Reply:
I think just giving the message object would be enough hmmm...